builder/dockerfile/parser/parser.go

```diff
@@ -101,19 +103,24 @@ func parseLine(line string) (string, *Node, error) {
 // Parse is the main parse routine.
 // It handles an io.ReadWriteCloser and returns the root of the AST.
 func Parse(rwc io.Reader) (*Node, error) {
+	currentLine := 0
 	root := &Node{}
+	root.StartLine = currentLine
```

Comment:
Actually the set of StartLine will need to be event lower if we really do want to have it skip blank lines.  Maybe that's not worth it? Dunno, we'll have to see how it looks.